README.md

```diff
@@ -141,13 +141,13 @@ packs with the chunks in your view, which creates html tags for all the chunks.
 The result looks like this:
 
 ```erb
-<%= javascript_pack_tag 'calendar', 'map' %>
+<%= javascript_pack_tag 'calendar', 'map', 'data-turbolinks-track': 'reload', defer: true %>
 
-<script src="/packs/vendor-16838bab065ae1e314.js" data-turbolinks-track="reload"></script>
-<script src="/packs/calendar~runtime-16838bab065ae1e314.js" data-turbolinks-track="reload"></script>
-<script src="/packs/calendar-1016838bab065ae1e314.js" data-turbolinks-track="reload"></script>
-<script src="/packs/map~runtime-16838bab065ae1e314.js" data-turbolinks-track="reload"></script>
-<script src="/packs/map-16838bab065ae1e314.js" data-turbolinks-track="reload"></script>
+<script src="/packs/vendor-16838bab065ae1e314.js" data-turbolinks-track="reload" defer="defer"></script>
+<script src="/packs/calendar~runtime-16838bab065ae1e314.js" data-turbolinks-track="reload" defer="defer"></script>
+<script src="/packs/calendar-1016838bab065ae1e314.js" data-turbolinks-track="reload" defer="defer"></script>
+<script src="/packs/map~runtime-16838bab065ae1e314.js" data-turbolinks-track="reload" defer="defer"></script>
+<script src="/packs/map-16838bab065ae1e314.js" data-turbolinks-track="reload" defer="defer"></script>
 ```
 
 **Important:** Pass all your pack names as multiple arguments, not multiple calls, when using **`javascript_pack_tag`** and the **`stylesheet_pack_tag`**. Otherwise, you will
```

Comment:
```suggestion
**Important:** In order to avoid hard to debug rendering issues, 
it's important that you pass all your pack names as multiple string **arguments**, not multiple calls, 
when using **`javascript_pack_tag`** and the **`stylesheet_pack_tag`**. Otherwise, you will
```

Reply:
This is a good change.

or (humorously)

```
**Important:** In order to avoid undue **suffering** with hard to debug rendering issues, 
```